ReactSkia/pluginfactory/RnsPluginFactory.cpp

```diff
@@ -31,6 +31,24 @@ RnsPluginFactory::RnsPluginFactory(){
     RNS_LOG_ERROR(" Platform Application Manager Plugin Loading Error :  " << ex.what());
   }
 #endif // RNSP_APP_MANAGER
+
+#if ENABLE(RNSP_PLATFORM_MANAGER)
+  RNS_LOG_INFO("========== Loading Platform Manager Plugin : " << RNSP_PLATFORM_MANAGER_PLUGIN << " for RNS ==========");
+  try {
+    platformPluginHandler_ = dlopen(RNSP_PLATFORM_MANAGER_PLUGIN, RTLD_NOW);
+    if (nullptr == platformPluginHandler_) {
+      RNS_LOG_ERROR("Could not Load Platform Manager Plugin : " << RNSP_PLATFORM_MANAGER_PLUGIN);
+      Check_dlerror();
+    }
+    platformManagerCreator_ = reinterpret_cast<RNSPlatformManagerInterface_t>(dlsym(platformPluginHandler_, "RnsPluginPlatformCreate"));
```

Comment:
Do we need to continue loading the sym , when the library loading itself fails ??

Reply:
Check_dlerror()  it throw Exception.